guide/07-working-with-bigdata/01-introduction.ipynb

```diff
@@ -180,6 +180,13 @@
    "source": [
```

Comment:
<p><strong>GeoAnalytics Server uses</strong></p><p>GeoAnalytics Server is helpful when you find that your current GIS analyses aren’t processing your data fast enough. It accelerates traditional workflows so you can get results quicker. GeoAnalytics Server is also helpful when you have large datasets and you need to analyze them spatially. GeoAnalytics Server is a good solution for the following:</p><ul><li>Your existing tools and workflows aren’t processing your data fast enough. </li><li>Your data is growing and you need a better way of managing and analyzing it. </li><li>You need to transform your data into something more manageable to use in other GIS analyses (for example using ArcGIS Pro analysis tools). </li><li>Your data has a lot of noise and you want to explore it to identify important points. </li><li>You want to use spatial statistical analysis and machine learning tools suitable for large datasets.</li></ul><p> </p><p><strong>Examples of analysis with ArcGIS GeoAnalytics Server</strong></p><ul><li>As a crime analyst, you can understand the location and time of crimes in your state, as well as the proximity of crimes to areas of interest, such as events, police stations, and city centers. Related tools are Aggregate Points and Join Features.</li><li>As a manager at a state Department of Transportation, you can analyze decades of traffic and crash data to determine the interstates with the most incidents. You can also analyze when certain vehicles were speeding and braking and correlate them with the locations of vehicular accidents. Related tools are Find Point Clusters and Reconstruct Tracks.</li><li>As a water utility technician, you can sort through work orders for leaks and join them to a dataset of soil types to determine if leaks have occurred in areas where there is particularly corrosive soil. Related tools are Create Space Time Cube and Find Hot Spots.</li><li>As a city GIS analyst, you can use ArcGIS GeoEvent Server to ingest GPS data on all city vehicles, like public works vehicles and snow plows, and see where the vehicles have travelled, areas that have less coverage, and instances where vehicles exceeded the speed limit. Related tools are Reconstruct Tracks, Aggregate Points, and Detect Incidents.</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>There are some changes in the second section's bullet points that still need to be made.</p>